packages-ts/gauntlet-cosmos-contracts/test/utils.ts

```diff
@@ -0,0 +1,201 @@
+import DeployFlags from '../src/commands/contracts/flags/deploy'
+import DeployValidator from '../src/commands/contracts/deviation_flagging_validator/deploy'
+import DeployOCR2 from '../src/commands/contracts/ocr2/deploy'
+
+export type DeployResponse = {
+  responses: {
+    tx: string
+    contract: string
+  }[]
+}
+export const ONE_TOKEN = BigInt('1000000000000000000')
+
+export const MNEMONIC =
+  'surround miss nominee dream gap cross assault thank captain prosper drop duty group candy wealth weather scale put'
+export const NODE_URL = 'http://127.0.0.1:26657'
+export const DEFAULT_GAS_PRICE = '0.025ucosm'
+export const NETWORK = 'local'
+
+export const TIMEOUT = 180000
+
+export const CMD_FLAGS = {
+  network: NETWORK,
+  mnemonic: MNEMONIC,
+  nodeURL: NODE_URL,
+  gauntletTest: true,
+  defaultGasPrice: DEFAULT_GAS_PRICE,
+}
+
+/// Deploy Commands
+export const deployOCR2 = async (params: { [key: string]: any }) => {
+  const cmd = new DeployOCR2(
+    {
+      ...CMD_FLAGS,
+      ...params,
+    },
+    [],
+  )
+  const result = await cmd.run()
+  return result['responses'][0]['contract'] as string
+}
+
+export const deployAC = async () => {
+  const cmd = new DeployAC(
+    {
+      ...CMD_FLAGS,
+    },
+    [],
+  )
+  const result = await cmd.run()
+  return result['responses'][0]['contract'] as string
+}
+
+export const deployLink = async () => {
+  const cmd = new DeployLink(
+    {
+      ...CMD_FLAGS,
+    },
+    [],
+  )
+  await cmd.invokeMiddlewares(cmd, cmd.middlewares)
+  const result = ((await cmd.execute()) as unknown) as DeployResponse
+  return result.responses[0].contract
+}
+
+export const deployFlags = async (raiseAC: string, lowerAC: string) => {
+  const cmd = new DeployFlags(
+    {
+      ...CMD_FLAGS,
+      raisingAccessController: raiseAC,
+      loweringAccessController: lowerAC,
+    },
+    [],
+  )
+  const result = await cmd.run()
+  return result['responses'][0]['contract'] as string
+}
+
+export const deployValidator = async (flagsAddr: string, threshold: string) => {
+  const cmd = new DeployValidator(
+    {
+      ...CMD_FLAGS,
+      flags: flagsAddr,
+      flaggingThreshold: threshold,
+    },
+    [],
+  )
+  const result = await cmd.run()
+  return result['responses'][0]['contract'] as string
+}
+
+/// Setup and Teardown Helpers
+
+export const endWasmd = async () => {
+  if (process.env.SKIP_WASMD_SETUP) {
+    return
+  }
+  const downScript = path.join(__dirname, '../../../scripts/wasmd.down.sh')
+  execSync(`${downScript}`)
+}
+
+const WASMD_ACCOUNTS = path.join(__dirname, './devAccounts.json')
+
+/**
+ * Initializes Wasmd and Contracts, unless SKIP_WASMD_SETUP=true
+ * Can save time for debugging / local testing if Wasmd and base contracts have been setup already
+ *
+ * Will read initialized account addresses from local file system if SKIP_WASMD_SETUP=true
+ *
+ * @returns {string[]} Initialized account addresses
+ */
+export const maybeInitWasmd = async () => {
```

Comment:
copied from the other closed PR: 'maybeInitWasmd' seems kinda confusing semantically (is wasmd initialized or not?)- perhaps just 'initWasmd' would suffice?

Reply:
Sounds good, I can change it. I originally but "maybe" because if SKIP_WASMD=true then wasmd is not initialized. But it might not be clear from the naming